sdk/python/README.md

```diff
@@ -129,6 +129,11 @@ When code changes are required to static helper methods in `kfp.compiler` the "o
 their respective modules in `kfp_tekton.compiler` and added to the _monkey patch_ which dynamically replaces the code in
 the `kfp` at runtime.
 
+**Note**: As of May 2020, the _monkey patch_ was no longer needed and removed since all of the _patched_ methods were
```

Comment:
I think we should move the monkey patching section to the bottle and mark it as deprecated. This way the users won't get confused if they just reading from middle of the page.

Reply:
Okay. I moved the detailed description and code snippets for the _monkey patch_ to the bottom of the developer guide:
https://github.com/ckadner/kfp-tekton/blob/remove_monkey_patch/sdk/python/README.md#adding-new-code

But I do want to leave the main paragraph about it in the middle of the doc for future reference since it is one of two main ways to inject code into the KFP compiler and it may not be obvious to other developers how to implement it, or, that it even is a possibility.

Reply:
Thanks @ckadner